app/views/manage/finishers/show.html.haml

```diff
@@ -51,8 +51,11 @@
       .country
         = ISO3166::Country[@finisher.country]&.iso_short_name
     .page-section.mb-4
-      %b Workplace Match?
+      %b Workplace Match Donations?
       = @finisher.has_workplace_match == nil ? "I don't know" : @finisher.has_workplace_match ? "Yes" : "No"
+    .page-section.mb-4
+      %b Workplace Match Volunteer Hours?
+      = @finisher.has_volunteer_time_off? ? "Yes" : "No"
```

Comment:
Should we call out the nil/unknown value here? It effectively a no unless we want to specifically ask them so I could go either way.